composer.json

```diff
@@ -12,10 +12,10 @@
     ],
     "require": {
         "php": "^7.1.0",
+        "psr/http-factory": "^1.0",
+        "sensio/framework-extra-bundle": "^5.5",
```

Comment:
is the world ready to upgrade?

https://symfony.com/releases

Symfony 3.4 is still supported as LTS till November 2021.

Reply:
I didn't think about that.
I guess you are right, I will remove it. 👍